tests/functional/install/basic.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+global_setup() {
+
+    create_testworking_dir
+    sudo qemu-img create -f raw "$TESTWORKINGDIRECTORY"/testimgfile.img 5G
+    loopbackdevice=$(sudo losetup --partscan --find --show "$TESTWORKINGDIRECTORY"/testimgfile.img)
+    export loopbackdevice
+
+}
+
+global_teardown() {
+    for i in $(losetup -O Name -n); do
+        sudo losetup -d "$loopbackdevice"
+    done
+
+    clean_testworking_dir
+}
+
+@test "INSTALL001: Basic Install" {
+
+ run sudo sh -c "$CLR_INSTALLER_EXE -c $TESTSCRIPTS/basic.yaml -b installer:${loopbackdevice}"
+
+ command_exit_ok
+
+}
```

Comment:
Please add a newline
What editor are you using?

Reply:
Done